packages/fuels-core/src/codec/abi_decoder.rs

```diff
@@ -384,91 +385,91 @@ mod tests {
-        //     y: Bar,
-        // }
-        //
-        // struct Bar {
-        //     a: bool,
-        //     b: u8[2],
-        // }
-
-        // fn: long_function(Foo,u8[2],b256,str[3],str)
-
-        // Parameters
-        let fields = vec![
-            ParamType::U16,
-            ParamType::Struct {
-                fields: vec![
-                    ParamType::Bool,
-                    ParamType::Array(Box::new(ParamType::U8), 2),
-                ],
-                generics: vec![],
-            },
-        ];
-        let nested_struct = ParamType::Struct {
-            fields,
-            generics: vec![],
-        };
-
-        let u8_arr = ParamType::Array(Box::new(ParamType::U8), 2);
-        let b256 = ParamType::B256;
-        let s = ParamType::StringArray(3);
-        let ss = ParamType::StringSlice;
-
-        let types = [nested_struct, u8_arr, b256, s, ss];
-
-        let bytes = [
-            0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xa, // foo.x == 10u16
-            0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, // foo.y.a == true
-            0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, // foo.b.0 == 1u8
-            0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x2, // foo.b.1 == 2u8
-            0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, // u8[2].0 == 1u8
-            0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x2, // u8[2].0 == 2u8
-            0xd5, 0x57, 0x9c, 0x46, 0xdf, 0xcc, 0x7f, 0x18, // b256
-            0x20, 0x70, 0x13, 0xe6, 0x5b, 0x44, 0xe4, 0xcb, // b256
-            0x4e, 0x2c, 0x22, 0x98, 0xf4, 0xac, 0x45, 0x7b, // b256
-            0xa8, 0xf8, 0x27, 0x43, 0xf3, 0x1e, 0x93, 0xb, // b256
-            0x66, 0x6f, 0x6f, 0x00, 0x00, 0x00, 0x00, 0x00, // str[3]
-            0x54, 0x68, 0x69, 0x73, 0x20, 0x69, 0x73, 0x20, // str data
-            0x61, 0x20, 0x66, 0x75, 0x6c, 0x6c, 0x20, 0x73, // str data
-            0x65, 0x6e, 0x74, 0x65, 0x6e, 0x63, 0x65, // str data
-        ];
-
-        let decoded = ABIDecoder::default().decode_multiple(&types, &bytes)?;
-
-        // Expected tokens
-        let foo = Token::Struct(vec![
-            Token::U16(10),
-            Token::Struct(vec![
-                Token::Bool(true),
-                Token::Array(vec![Token::U8(1), Token::U8(2)]),
-            ]),
-        ]);
-
-        let u8_arr = Token::Array(vec![Token::U8(1), Token::U8(2)]);
-
-        let b256 = Token::B256([
-            0xd5, 0x57, 0x9c, 0x46, 0xdf, 0xcc, 0x7f, 0x18, 0x20, 0x70, 0x13, 0xe6, 0x5b, 0x44,
-            0xe4, 0xcb, 0x4e, 0x2c, 0x22, 0x98, 0xf4, 0xac, 0x45, 0x7b, 0xa8, 0xf8, 0x27, 0x43,
-            0xf3, 0x1e, 0x93, 0xb,
-        ]);
-
-        let ss = Token::StringSlice(StaticStringToken::new(
-            "This is a full sentence".into(),
-            None,
-        ));
-
-        let s = Token::StringArray(StaticStringToken::new("foo".into(), Some(3)));
-
-        let expected: Vec<Token> = vec![foo, u8_arr, b256, s, ss];
-
-        assert_eq!(decoded, expected);
-        Ok(())
-    }
+    // #[test]
```

Comment:
Does this need to be updated or removed?

Reply:
Why was this commented out?

Reply:
Fixed.